### PR TITLE
Release v11.23.0

### PR DESCRIPTION
**Pull request by stevehobbsdev**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
+## [v11.23.0](https://github.com/auth0/lock/tree/v11.23.0) (2020-04-02)
+[Full Changelog](https://github.com/auth0/lock/compare/v11.22.5...v11.23.0)
+
+
+**Added**
+- Add invalidHint to phoneNumberInput (fix issue #1836) [\#1837](https://github.com/auth0/lock/pull/1837) ([blankg](https://github.com/blankg))
+
+**Fixed**
+- [SDK-1413] Re-added scrollbars and fixed password strength popup clip issue [\#1839](https://github.com/auth0/lock/pull/1839) ([stevehobbsdev](https://github.com/stevehobbsdev))
+- add translate 'social_signup_needs_terms_acception' for japanese [\#1835](https://github.com/auth0/lock/pull/1835) ([mag-chang](https://github.com/mag-chang))
```
Comment (damieng) [inline CHANGELOG.md:12]:
```suggestion
- Add 'social_signup_needs_terms_acception' Japanese translation [\#1835](https://github.com/auth0/lock/pull/1835) ([mag-chang](https://github.com/mag-chang))
```

**Comment by lbalmaceda** [comment]
@stevehobbsdev do we want to bump the SDK version on this release as well?  https://github.com/auth0/lock/blob/master/package.json#L104

**Comment by stevehobbsdev** [comment]
@lbalmaceda Yes! Thanks for the reminder

**Comment by stevehobbsdev** [comment]
Raised here: https://github.com/auth0/lock/pull/1842

**Pull request merged**